.storybook/recipes/Filters/Filters.tsx

```diff
@@ -37,10 +36,167 @@ export const Filters = ({ placement, ...other }: Props) => {
+  };
+
+  /**
+   * Counts how many filters have been applied.
+   */
+  const filterCount = Object.values(checked).reduce((count: number, status) => {
+    if (status) return count + 1;
+    else return count;
+  }, 0);
+
+  const hasSelectedFilters = filterCount > 0;
+  const triggerText = hasSelectedFilters
+    ? `Filters (${filterCount})`
+    : 'Filters';
+
+  const checkboxes = (
+    <FiltersCheckboxField legend="Filters Segment 1">
+      <Checkbox
+        checked={checked.soup}
+        label="Soup"
+        onChange={() => onCheckboxChange('soup')}
+      />
+      <Checkbox
+        checked={checked.salad}
+        label="Salad"
+        onChange={() => onCheckboxChange('salad')}
+      />
+      <Checkbox
+        checked={checked.sandwich}
+        label="Sandwich"
+        onChange={() => onCheckboxChange('sandwich')}
+      />
+    </FiltersCheckboxField>
+  );
+
+  const foods = [
+    {
+      name: 'Cereal',
+      isSoup: true,
+      isSalad: true,
+      isSandwich: false,
+    },
+    {
+      name: 'Fried Rice',
+      isSoup: false,
+      isSalad: true,
+      isSandwich: false,
+    },
+    {
+      name: 'Hot Dog',
+      isSoup: false,
+      isSalad: false,
+      isSandwich: true,
+    },
+    {
+      name: 'Mashed Potatoes',
+      isSoup: true,
+      isSalad: true,
+      isSandwich: false,
+    },
+    {
+      name: 'Nigiri',
+      isSoup: false,
+      isSalad: true,
+      isSandwich: true,
+    },
+    {
+      name: 'Oatmeal',
+      isSoup: true,
+      isSalad: false,
+      isSandwich: false,
+    },
+    {
+      name: 'Soup in Bread Bowl',
+      isSoup: true,
+      isSalad: false,
+      isSandwich: true,
+    },
+    {
+      name: 'Sloppy Joe',
+      isSoup: true,
+      isSalad: true,
+      isSandwich: true,
+    },
+  ];
+
+  const filteredFoods = foods
+    .filter((food) => (checked.salad ? checked.salad === food.isSalad : true))
+    .filter((food) => (checked.soup ? checked.soup === food.isSoup : true))
+    .filter((food) =>
+      checked.sandwich ? checked.sandwich === food.isSandwich : true,
+    );
+
+  const foodTable = (
+    <Table>
+      <Table.Header>
+        <Table.Row variant="header">
+          <Table.HeaderCell className="w-1/4">Food</Table.HeaderCell>
+          <Table.HeaderCell className="text-center w-1/4">
+            Soup
+          </Table.HeaderCell>
+          <Table.HeaderCell className="text-center w-1/4">
+            Salad
+          </Table.HeaderCell>
+          <Table.HeaderCell className="text-center w-1/4">
+            Sandwich
+          </Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        {filteredFoods.map((food) => (
+          <Table.Row key={'table-row-' + food.name}>
+            <Table.Cell>{food.name}</Table.Cell>
+            <Table.Cell className="text-center w-1/4">
+              {(food.isSoup && (
+                <Icon name="star" purpose="informative" title="is soup" />
```

Comment:
```suggestion
                🍜
```

Reply:
lol j/k the stars are fine

Reply:
I actually did want to use emojis but could have accessibility issues and didn't want to encourage that